.github/workflows/02-release.yaml

```diff
@@ -0,0 +1,69 @@
+name: 02-create_release
+
+on:
+  push:
+    tags:
+    - 'v*.*.*-rc.*'
+
+jobs:
+  retag:
+    outputs:
+      NEW_TAG: ${{ steps.tag-calculator.outputs.NEW_TAG }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: tag-calculator
+        uses: ./.github/actions/tag-action
+        with:
+          SUB_STRING: "-rc"
+
+  binary-build:
+    needs: [retag]
+    env:
+      CUSTOMER: ${{ secrets.CUSTOMER }}
+      USERNAME: ${{ secrets.USERNAME }}
+      PASSWORD: ${{ secrets.PASSWORD }}
+      CLIENT_ID: ${{ secrets.CLIENT_ID_PROD }}
+      SECRET_KEY: ${{ secrets.SECRET_KEY_PROD }}
+      REGISTRY_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
+      REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
+    if: ${{ env.CUSTOMER && 
```

Comment:
[TODO]: We should have a step named `run-binary-build`, that step should store the output of the `if` statement so this step will look cleaner